### ISSUE TITLE
Tabr overwrites a CMD+Shift+T from another package. How can I change tabrs keymaps?

### ISSUE THREAD

**Issue by A5308Y** [mghweb/SublimeTabr#1]
Hi and thanks for tabr!

After installing I don't see a keymap file where I could change the default keymap. How can I remove the CMD+Shift+T shortcut? I want to use that for another package.

Thanks!

**Comment by mghdotdev** [comment]
Hello, I am glad you like it!

Open up your user keybindings file located at "Preferences -> Keybindings - User".

Paste in this code and change the key combination:
`{ "keys": [ "ctrl+shift+t" ], "command": "tabr" }`

If the file is blank paste this in and change the key combination:

```
[
    { "keys": [ "ctrl+shift+t" ], "command": "tabr" }
]
```

Let me know if you have any more questions!

**Comment by A5308Y** [comment]
Hi @mghweb!
Thanks for answering so quickly!
The method you provided didn't work for me. Instead both CMD+Shift+T and whatever the new combination I chose now call tabr. I'm used to packages having their own editable keybindings (Preferences -> Package Settings -> [Name of the package] -> Key Bindings - User).
I don't know if this is unnecessary if there's only one keybinding, but it seems this could help here.
I tried to find a settings file for tabr among the other sublime package settings files, but I couldn't I only see the tabr binary...

**Comment by mghdotdev** [comment]
Hello, I have added a Preferences menu with the Keybindings file. The update should be pushed through package control shortly.

Let me know if this fixes it for you!

**Comment by A5308Y** [comment]
Hi again!
I've updated tabr and found the menu entries. 
It didn't solve my problem in itself, but it put me on the right path. Thanks to your menu entry I knew where the default settings-file was expected. I created it and removed the keymap. Now everything works! Yeah!
I might have to do this again after the next tabr udpate, but it'll do.
And BTW "Keybindings - User" just seems to be a link to my default keymap...

Thanks again!
